css/lavish-bootstrap.css

```diff
@@ -717,7 +717,7 @@ pre code {
 .col-sm-12,
 .col-md-1,
 .col-md-2,
-.col-md-3,
+.col-md-3
```

Comment:
missing comma, think this is causing the sidebar to over extend into the page
![screenshot from 2017-09-12 13-01-03](https://user-images.githubusercontent.com/11265811/30341183-bc8a0b22-97ba-11e7-9135-fc4f643b6ea1.png)

just happen within the hour.

Reply:
I checked in a correction for this.... See if that fixes it.